examples/audio/librispeech.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Is it new code or copy-paste from audio repo? If latter, can we please put link to it.